automation/id_rsa_nmstate

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
How about using https://docs.travis-ci.com/user/encryption-keys ?

Reply:
It does not work for external PRs.